src/components/VmsPageHeader/UserMenu.js

```diff
@@ -28,7 +28,7 @@ const UserMenu = ({ config, onLogout, onAdministratorFilterClick }) => {
   } else {
     let checkbox = null
     if (config.get('administrator')) {
-      checkbox = (<li><a><Checkbox checked={config.get('filter')} onClick={onAdministratorFilterClick} label='Filter VM' /></a></li>)
+      checkbox = (<li><a><Checkbox checked={config.get('filter')} onClick={onAdministratorFilterClick} label='Use Admin Permissions' /></a></li>)
```

Comment:
so, after a discussion with @michalskrivanek it will be better to remove this checkbox completely and just send filter: true/false according to the permissions of the user.